app/src/main/java/mozilla/lockbox/extensions/ViewNode+.kt

```diff
@@ -0,0 +1,64 @@
+package mozilla.lockbox.extensions
+
+import android.annotation.TargetApi
+import android.app.assist.AssistStructure
+import android.os.Build
+
+@TargetApi(Build.VERSION_CODES.O)
+fun AssistStructure.ViewNode.dump(): String {
+    val sb = StringBuilder("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
+    return dumpNode(sb).toString()
+}
+
+@TargetApi(Build.VERSION_CODES.O)
+private fun AssistStructure.ViewNode.dumpNode(sb: StringBuilder = StringBuilder()): StringBuilder {
```

Comment:
The name `AssistStructure` confuses me. This is only for assisting autofill attributes?